packages/mds-db/processors.ts

```diff
@@ -12,6 +12,12 @@ import schema from './schema'
 import { vals_sql, cols_sql, vals_list, logSql, SqlVals } from './sql-utils'
 import { getWriteableClient, makeReadOnlyQuery } from './client'
 
+export async function getVehicleType(device_id: UUID): Promise<Array<{ [vehicle_type: string]: VEHICLE_TYPE }>> {
+  const vals = new SqlVals()
+  const query = `SELECT DISTINCT vehicle_type FROM reports_device_states WHERE device_id = ${vals.add(device_id)}`
+  return makeReadOnlyQuery(query, vals)
+}
+
```

Comment:
This seems like an odd way to get the vehicle type for a specific `device_id`. Why not just get it by selecting a single row from the `devices` table?